seaborn/categorical.py

```diff
@@ -2423,26 +2461,30 @@ def boxenplot(
     showfliers : bool, optional
         If False, suppress the plotting of outliers.
     {ax_in}
+    box_kws: key, value mappings
+        Other keyword arguments are passed through to
+        :meth:`matplotlib.patches.Rectangle`.
+    line_kws: key, value mappings
+        Other keyword arguments are passed through to
```

Comment:
Same comment as above, but please clarify that this pertains to the line denoting the median.